maintainers/README.md

```diff
@@ -72,18 +72,21 @@ As the Nix community is distributed globally, available time for synchronous com
 
 To keep discussions highly focused and make their results accessible, we will:
 
-- set meeting agendas on [the team's GitHub project board](https://github.com/orgs/NixOS/projects/15) from issues and pull requests
+- prepare meeting agendas as a pull requests on the [nix.dev repository](https://github.com/NixOS/nix.dev)
```

Comment:
@infinisil writes agendas as a Discourse thread, and the reply are the meeting notes